### PR TITLE
[id] Improve translation

### PR DESCRIPTION
**Pull request by itsdzl**
searching for credits command to change my tag, but im not find it

### CODE REVIEW

**Comment by itsdzl** [comment]
try it

File: id.hjson [hunk 1]
```diff
@@ -275,45 +275,45 @@
...
     "main": "<b>*</b>%s menepuk kepala %s<b>* :V</b>"
     "CalsiBot": "<i>rawr</i> <b>xDddd</b>"
   }
 
   "comfy.ruffle": {
-    "self": "%s<b> mengacak-acak rambutnya sendiri... Astaga. 😐</b>"
-    "main": "<b>*</b>%s mengacak rambut %s<b>* :&</b>"
-    "CalsiBot": "<i>Kamu mengacak-acak rambutku ... tidak ... di sana.</i> 😳"
+    "self": "%s<b> mengacak-acak rambutnya sendiri... yaampun. 😐</b>"
+    "main": "<b>*</b>%s mengacak rambut %s<b>* :&</b> 😝"
```
Comment (CalsiBotDev) [inline id.hjson:309]:
```suggestion
    "main": "<b>*</b>%s mengacak rambut %s<b>* :&</b>"
```

File: id.hjson [hunk 2]
```diff
@@ -810,7 +809,7 @@
   #
   #          This list of names should be in order of contribution size and frequency, taking into account age of last
   #          contribution.
-  "language.credit.activeContributors": "<a href=\"t.me/urdzL\">dzL</a>"
+  "language.credit.activeContributors": "<a href=\"t.me/izels\">dzL</a>"
```
Comment (CalsiBotDev) [inline id.hjson:812]:
```suggestion
  "language.credit.activeContributors": "<a href=\"tg://user?id=5966163635\">dzL</a>"
```

File: id.hjson [hunk 3]
```diff
@@ -275,45 +275,45 @@
...
     "CalsiBot": "<i>rawr</i> <b>xDddd</b>"
   }
 
   "comfy.ruffle": {
-    "self": "%s<b> mengacak-acak rambutnya sendiri... Astaga. 😐</b>"
-    "main": "<b>*</b>%s mengacak rambut %s<b>* :&</b>"
-    "CalsiBot": "<i>Kamu mengacak-acak rambutku ... tidak ... di sana.</i> 😳"
+    "self": "%s<b> mengacak-acak rambutnya sendiri... yaampun. 😐</b>"
+    "main": "<b>*</b>%s mengacak rambut %s<b>* :&</b> 😝"
+    "CalsiBot": "<i>Anda seharusnya mengacak -acak rambut di kepalaku ... jangan ... di sana. 😳</i> 😑"
```
Comment (CalsiBotDev) [inline id.hjson:310]:
```suggestion
    "CalsiBot": "<i>Anda seharusnya mengacak -acak rambut di kepalaku ... jangan ... di sana.</i> 😳"
```

**Review by CalsiBotDev — CHANGES_REQUESTED** [review]

**Review by CalsiBotDev — APPROVED** [review]

**Pull request merged**